solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -87,6 +87,9 @@ public void filter(DocumentIndexingPackage dip) throws IndexingException {
         if (dip.getDocument().getTitle() == null) {
             idb.setTitle(getAlternativeTitle(dip));
         }
+        if (dip.getDocument().getDateCreated() == null) {
+            idb.setDateCreated(idb.getDateAdded());
```

Comment:
is getDateAdded() guaranteed to return a non-null value here? if not, how are we handling a null value?

Reply:
yes, date added should be always be non-null since it is a generated field from fedora.  If it weren't available, then the date created field would just get cleared.